### DISCUSSION TITLE
FindReplaceBar class.

### DISCUSSION THREAD

**Discussion by ghost** [godotengine/godot-proposals#7075] [GUI]
The FindReplaceBar class in the Editor source could be a useful class to expose that would offer a basic search/replace text operations interface.  The "text" could be supplied via another Control, eg. a TextEdit allowing it to be extended as necessary.

**Comment by AThousandShips** [comment]
This class is only available in the editor, and isn't general, it's specific to the editor and can't be used generally

**Comment by ghost** [comment]
Can't it be amended for generality ?

**Reply by AThousandShips** [reply]
Not that I know, also please use the thread function, it's very inefficient when you comment separately like this instead of replying to my comments

**Comment by ghost** [comment]
That's a shame, because it could be useful.
